lib/jekyll-relative-links/generator.rb

```diff
@@ -21,16 +21,52 @@ def initialize(site)
     end
 
     def generate(site)
+      # Get any configuration specified
+      # Defaults mean no change in behaviour
+      jrl_config = site.config["jekyll_relative_links"] || {}
+      @config = {}
+      @config["collections"] = jrl_config["collections"] || []
+      @config["process_all_collections"] = jrl_config["process_all_collections"] || false
+      @config["verbose"] = jrl_config["verbose"] || 0
+
       @site    = site
       @context = context
 
       site.pages.each do |page|
         next unless markdown_extension?(page.extname)
         replace_relative_links!(page)
       end
+
+      site.collections.each do |collection|
+        label = collection[1].label
+        if @config["process_all_collections"] || @config["collections"].include?(label)
```

Comment:
I'd make a `should_process_collection?` or similar helper method, that always returns true if set to process all collections, otherwise, it checks the config for the collection to help keep this readable. This should also check writability here to save cycles.